ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -155,6 +165,123 @@ static os_log_type_t KextLogLevelAsOSLogType(KextLog_Level level)
     }
 }
 
+static void LogPanics()
+{
+    DIR *dr = opendir(PanicLogDirectory);
+    if (dr == NULL)
+    {
+        ostringstream errorMessage;
+        errorMessage << "Unable to open " << PanicLogDirectory << ", errno=" << errno << ", errorstr=" << strerror(errno);
+        LogDaemonError(errorMessage.str());
+    }
+
+    // Look up the last run time
+    long lastRunTimestamp = 0;
+    if (DoesPathExist(PanicLogTimestampFile))
```

Comment:
There's no need to call `DoesPathExist` here as `GetFileModifiedTime` will return 0 if the file does not exist.